documentation/api/introduction.rst

```diff
@@ -139,3 +144,19 @@ Here is a client-side code example in Python (this merely prints out the depreca
                 print(f"Your request to {url} returned a sunset warning. Sunset: {content}")
             elif header == "Link" and ('rel="deprecation";' in content or 'rel="sunset";' in content):
                 print(f"Further info is available: {content}")
+
+.. _api_deprecation_hosts:
+
+FlexMeasures hosts
+^^^^^^^^^^^^^^^^^^
+
+When upgrading to a FlexMeasures version that sunsets an API version, clients will receive ``HTTP status 410 (Gone)`` responses when calling corresponding endpoints.
+After upgrading to one of the next FlexMeasures versions, they will receive ``HTTP status 404 (Not Found)`` responses.
+
+Hosts should not expect every client to monitor response headers and proactively upgrade to newer API versions.
+Please make sure that your users have upgraded before you upgrade to a FlexMeasures version that sunsets an API version.
+You can do this by checking your server logs for warnings about users who are still calling deprecated endpoints.
+
+In case you have users that haven't upgraded yet, and would still like to upgrade FlexMeasures, you can.
```

Comment:
I fee it's unclear at what time this is available.

This is a setting which has an effect one version after a sunset (in this case 0.13), after which we should make in ineffectual (old code has got to go).
Or should it in principle even be active right after deprecation?

We need more clarity here.

Reply:
Good idea to allow blackout tests already when first announcing the deprecation. I'll add that, and also clarify the rest.